Shared/Views/SearchTab/SearchView.swift

```diff
@@ -21,22 +21,14 @@ struct SearchView: View {
     var body: some View {
         NavigationViewIOS(viewBuilder: {
             ScrollView(.vertical) {
-                TextField("Search", text: $searchText) { changed in
-                    print("changed: \(changed), text: \(searchText)")
-                } onCommit: {
-                    print("onCommit, text: \(searchText)")
-                    if let session = store.state.crState.session, searchText.trimmingCharacters(in: .whitespacesAndNewlines).count > 0 {
-                        store.dispatch(action: CRActions.Autocomplete(sessionId: session.id, q: searchText))
-                    }
-                }
-                .padding(7)
-                #if os(iOS)
-                .background(Color(.systemGray6))
-                .cornerRadius(8)
-                .padding(.horizontal, 10)
-                #endif
                 CRSearchResult(result: result)
             }
+            .searchable(text: $searchText)
```

Comment:
I looked into Apple Fruta app.
`Text(suggestion.name).searchCompletion(suggestion.name)` seems not work for the current `ScrollView>VStack`.
Will consider change to `List` later.